canonical/snapcraft.io
# Convert brand store navigation to TypeScript

## Done
Converted the brand store navigation to TypeScript

## How to QA
- Go to https://snapcraft-io-4307.demos.haus/admin
- Check that the side navigation renders and works without errors
